docs/source/contents/kubernetes/storage-secrets/index.md

```diff
@@ -0,0 +1,147 @@
+# Storage Secrets
+
+Inference artifacts referenced by Models can be stored in any of the storage backends supported by [Rclone](https://rclone.org/).
+This includes local filesystems, AWS S3, and Google Cloud Storage (GCS), among others.
+Configuration is provided out-of-the-box for public GCS buckets, which enables the use of Seldon-provided models like in the below example:
+
+```{literalinclude} ../../../../../samples/models/sklearn-iris-gs.yaml 
+:language: yaml
+```
+
+This configuration is provided by the Kubernetes `secret` `seldon-rclone-gs-public`.
```

Comment:
```suggestion
This configuration is provided by the Kubernetes Secret `seldon-rclone-gs-public`.
```

this will allow to avoid two code-blocks to each other and seems more along how official [documentation](https://kubernetes.io/docs/concepts/configuration/secret/) references K8s resources.